views/profile.handlebars

```diff
@@ -1,31 +1,6 @@
-<!DOCTYPE html>
-<html>
-<title>PetStterz Profile</title>
-<meta charset="UTF-8">
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
-<link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Roboto'>
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css"/>
-<script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
-<style>
-html,body,h1,h2,h3,h4,h5,h6 {font-family: "Roboto", sans-serif}
-</style>
-<body class="w3-light-grey">
-    <div class="container">
-        <!-- Static navbar -->
-        <nav class="navbar navbar-default">
-          <div class="container-fluid">
-            <div id="navbar" class="navbar-collapse collapse">
-              <ul class="nav navbar-nav">
-                <li class="active"><a href="./profile.handlebars">Home</a></li>
-              </ul>
-            </div><!--/.nav-collapse -->
-          </div><!--/.container-fluid -->
+<div class="container">
+<a href src="/socket/index.html">Socket Chat App</a>
```

Comment:
Don't know what this is really, but thought it'd be easier to reference if I moved it to the `public` folder.  It's static, whatever it is, so it should work.